api-reference/10 UI Widgets/dxFileUploader/1 Configuration/uploadAbortedMessage.md

```diff
@@ -5,7 +5,7 @@ default: 'Upload cancelled'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The message displayed by the widget when the file upload is cancelled. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This option makes sense only if the [uploadMode](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode) option is set to "instantly".
```

Comment:
I'm not sure what you mean by "makes sense". Do you mean:

This option is only available if the [uploadMode](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode) option is set to "instantly".

Reply:
Yes, you right. "Available" sounds better.

They are both options.

Reply:
Thank you. Please apply my suggestion.

Reply:
Applied